components/shopify_partner/sources/new_app_relationship_events.mjs

```diff
@@ -0,0 +1,79 @@
+import common from "../common.mjs";
```

Comment:
Does this reference the correct file? `common.mjs` and the app file live two dirs up, right?